builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -117,7 +124,7 @@ node() {
 		def stackName = "${configLoader.INSTANCE_PREFIX}-${config['domain']}-${config['service']}-${environment_logical_id}"
 
 		def internalAccess = config['require_internal_access']
-		def roleARN = configLoader.AWS.ROLEID.replaceAll("/", "\\\\/")
+		def roleARN = config['iamRoleARN']
```

Comment:
No need of initializing def roleARN here. Use config['iamRoleARN'] instead of roleARN